### PR TITLE
OK-521  - Introduce configurable http timeout for non-deployment operations

### PR DESCRIPTION
**Pull request by cnicolov**
Introduce configuration option to set http timeout for all requests made to the Orka API except deploy - it has it's own configuration option for deploy timeout.

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/orka/OrkaCloud.java [hunk 1]
```diff
@@ -44,36 +44,46 @@
     private static final Logger logger = Logger.getLogger(OrkaCloud.class.getName());
     private static final int recommendedMinTimeout = 30;
     private static final int defaultTimeout = 600;
+    private static final int defaultHttpTimeout = 600;
```
Comment (ispasov) [inline src/main/java/io/jenkins/plugins/orka/OrkaCloud.java:47]: Let's reduce this to 300.
It should be more than enough

File: src/main/resources/io/jenkins/plugins/orka/OrkaCloud/help-httpTimeout.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<div>The time after which the request to the Orka API will timeout. Defaults to 600 seconds (10 minutes).</div>
```
Comment (ispasov) [inline src/main/resources/io/jenkins/plugins/orka/OrkaCloud/help-httpTimeout.html:1]: You can mention that it is for all operations, but Deploy

**Comment by radeva** [comment]
You can keep the good practice to add the Jira item number in the title ;) In this case it is OK-521

File: src/main/resources/io/jenkins/plugins/orka/OrkaCloud/help-httpTimeout.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<div>The time after which the request to the Orka API will timeout. This timeout is used for all HTTP operations but <b>deploy</b>. Defaults to 300 seconds (5 minutes).</div>
```
Comment (radeva) [inline src/main/resources/io/jenkins/plugins/orka/OrkaCloud/help-httpTimeout.html:1]:
```suggestion
<div>The time after which the request to the Orka API timeouts. This timeout is used for all HTTP operations but <b>deploy</b>. Defaults to 300 seconds (5 minutes).</div>
```

**Review by radeva — APPROVED** [review]

**Review by ispasov — APPROVED** [review]

**Pull request merged**